addons/fcs/functions/fnc_keyUp.sqf

```diff
@@ -62,15 +63,18 @@ private ["_movingAzimuth", "_posTarget", "_velocityTarget"];
 _movingAzimuth = 0;
 if (ACE_time - GVAR(ACE_time) > 1 and GVAR(ACE_time) != -1 and count _this < 3) then {
     // calculate speed of target
+    (getPos _vehicle) params ["_vehicleX", "_vehicleY", "_vehicleZ"];
```

Comment:
I realize it's not the purpose of the cleanup, but doing the calculations based on getPos probably breaks the target leading if the target is over water and the player is on land or vice versa. It would probably be better to do all the calculations ASL.